### PR TITLE
fix(OHI Elastic): Fixed errors in the installation doc

### PR DESCRIPTION
**Pull request by nbaenam**
Fixed errors in the [Elasticsearch monitoring integration](https://docs.newrelic.com/install/elastic/)

### CODE REVIEW

File: src/install/elasticsearch/whatsNext.mdx [hunk 1]
```diff
@@ -260,7 +260,7 @@ The Elasticsearch integration collects both Metrics and Inventory information. I
       </td>
 
       <td>
-        A way to further specify which cluster we are gathering data for, example:
+        A way to further specify which cluster we are gathering data, for example:
```
Comment (cbehera-newrelic) [inline src/install/elasticsearch/whatsNext.mdx:263]: Alternate: You can specify the cluster from which you are gathering data, for example, 'staging'.

**Review by cbehera-newrelic — DISMISSED** [review]
LGTM

File: src/install/elasticsearch/whatsNext.mdx (hunk 1 above)
Comment (nbaenam) [reply src/install/elasticsearch/whatsNext.mdx:263]: Done!

**Review by cbehera-newrelic — APPROVED** [review]
LGTM

**Pull request merged**